app/admin/reports/interval_data.rb

```diff
@@ -57,5 +57,6 @@ def scoped_collection
       column col
     end
     column assigns[:report_interval_cdr].aggregation, :aggregated_value
+    column :destination_rate_policy
```

Comment:
CDR_COLUMNS  already includes destination_rate_policy_id do we need to render it again ?